server_setup.md

```diff
@@ -0,0 +1,29 @@
+# Server Setup for creating a low privilege user
+#### Aim is to create such a user role that has the minimum privileges required to create a patient via an API call only.
+Please follow the steps listed below in order
+1. Log in to the server as an Admin.
+   * **username : admin**
+   * **password : Admin123**
```

Comment:
We're writing this for someone who has a working OpenMRS installation and who has (hopefully) changed the default username and password. Let's just leave step 1 at "log in as a user with administrative permissions"

Reply:
changes done